int32 energy
---
bool response
